src/main/java/pro/sky/collections1/service/EmployeeServiceImpl.java

```diff
@@ -89,4 +93,12 @@ public Map getMapEmployeesByDepartments() {
                 .collect(Collectors.groupingBy(Employee::getDepartment));
         return map;
     }
+    //Проверка вводимого текста (наличие только букв) и установка заглавной первой буквы
+    @Override
+    public String checkingInputTextData(String text) {
```

Comment:
по названию метод делает только проверку, стоит отразить в имени подготовку данных
```suggestion
    public String checkAndPrepareInputTextData(String text) {
```